superset-frontend/src/pages/AllEntities/index.tsx

```diff
@@ -114,13 +135,34 @@ function AllEntities() {
   };
   const items = [description, owner, lastModified];
 
+  const fetchTaggedObjects = () => {
+    setLoading(true);
+    fetchObjects(
+      { tags: tag?.name || '', types: null },
+      (data: TaggedObject[]) => {
+        const objects = { dashboard: [], chart: [], query: [] };
+        data.forEach(function (object) {
+          const object_type = object.type;
+          objects[object_type].push(object);
+        });
+        setObjects(objects);
+        setLoading(false);
+      },
+      (error: Response) => {
+        addDangerToast('Error Fetching Tagged Objects');
+      },
+    );
+  };
+
   useEffect(() => {
     // fetch single tag met
     if (tagId) {
+      setLoading(true);
       fetchSingleTag(
         tagId,
         (tag: Tag) => {
           setTag(tag);
+          setLoading(false);
         },
         (error: Response) => {
           addDangerToast(t('Error Fetching Tagged Objects'));
```

Comment:
you might need to add `setLoading(false);` here to prevent the spinner kept spinning